components/support/migration/src/main/java/mozilla/components/support/migration/AbstractMigrationService.kt

```diff
@@ -38,7 +35,7 @@ private const val TEMPORARY_NOTIFICATION_CHANNEL_NAME = "Migration"
 abstract class AbstractMigrationService : Service() {
     protected abstract val migrator: FennecMigrator
     protected abstract val store: MigrationStore
-    protected abstract val migrationActivity: Class<out AbstractMigrationProgressActivity>
+    protected abstract val migrationActivity: Class<out Activity>
```

Comment:
Don't we need two activity references here so that we can route to either the migration activity or the browser/onboarding?

Reply:
Ah, I see you decided to route to the decision activity instead. That should work too.

Reply:
Maybe we should rename this reference though? :)